### ISSUE TITLE
Schedule from program guide

### ISSUE THREAD

**Issue by bah1976** [MythTV-Clients/MythTV-Android-Frontend#74]
Unless I'm missing something, is it not possible to schedule a recording from the guide?  Can this be added as an enhancement request?

**Comment by pot8oe** [comment]
It is planned. Having some issues with the MythTV services API
AddRecordSchedual end point

-Tom

**Comment by dmfrey** [comment]
It is on the todo list.  We need to get editing the recording rules in
place first.  Then they will all use the same functions in the app.

On Tue, Nov 13, 2012 at 4:14 PM, bah1976 notifications@github.com wrote:

> Unless I'm missing something, is it not possible to schedule a recording
> from the guide? Can this be added as an enhancement request?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/MythTV-Clients/MythTV-Android-Frontend/issues/74.

## 

My Google Profile http://www.google.com/profiles/dmfrey

**Comment by billmeek** [comment]
Update: 0.27 appears to have the required enhancements
in AddRecordedSchedule..

**Comment by dmfrey** [comment]
Yes, we just need a dialog fragment that is reusable across recorded,
upcoming, guide. It was something I have been meaning to tackle.

I do think we should provide a warning for v25-26 as it needs to delete the
rule and add a new one, whereas v27 has an update endpoint.

**Comment by pot8oe** [comment]
Didn't I make an activity or fragment for this already? Might be able to
reuse the layout and convert to dialog fragment.

**Comment by andy-lawson** [comment]
This is the killer feature for me. I need to be able to schedule a recording here. Other than that - looking good!

**Comment by billmeek** [comment]
Thanks for the comment. This is scheduled for the 1.30 release.

**Comment by exboyracer2** [comment]
As above, this would be my main usage of the app... scheduling recordings when away from home.
looking forward to it!
